src/platforms/android/usage/advanced-usage.mdx

```diff
@@ -23,7 +23,7 @@ compileOptions {
 
 // ADD SENTRY ANDROID AS A DEPENDENCY
 dependencies {
-    // https://github.com/getsentry/sentry-android/releases
+    // https://github.com/getsentry/sentry-java/releases
     implementation 'io.sentry:sentry-android-core:{version}'
 }
```

Comment:
Lets revisit this.
Link to release pages are really bad.
We should have copy pastable stuff here. Until we get release registry up, do we do it manually